Linux arm build fails occasionally

See:

```

/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:2816:1: error: unknown escape sequence '\ ' [-Werror,-Wunknown-escape-sequence]
GCEnd_V1T_TRACEPOINT_INSTANCE(GCEnd_V1)
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:759:27: note: expanded from macro 'GCEnd_V1T_TRACEPOINT_INSTANCE'
    ModuleLoadUnload_V1_TRACEPOINT_ARGS,
                          ^~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:513:2: note: expanded from macro 'ACEPOINT_ARGS'
        _DEFINE_TRACEPOINT(provider, name, _TP_PARAMS(args))
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:354:4: note: expanded from macro '_DEFINE_TRACEPOINT'
                        _TP_EXTRACT_STRING(_args),                              \
                        ^~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:340:33: note: expanded from macro '_TP_EXTRACT_STRING'
#define _TP_EXTRACT_STRING(...) #__VA_ARGS__
                                ^~~~~~~~~~~~
<scratch space>:51:27: note: expanded from here
"const unsigned intT Count\ const unsigned inti Depth_ const unsigned shortt ClrInstanceID"
                          ^~
In file included from /__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/eventprovdotnetruntime.cpp:24:
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:2821:1: error: unknown escape sequence '\ ' [-Werror,-Wunknown-escape-sequence]
GCCreateSegmentT_TRACEPOINT_INSTANCE(GCCreateSegment)
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:608:10: note: expanded from macro 'GCCreateSegmentT_TRACEPOINT_INSTANCE'
TRACEPOINT_EVENT_INSTANCE(\
         ^~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:513:2: note: expanded from macro 'T_EVENT_INSTANCE'
        _DEFINE_TRACEPOINT(provider, name, _TP_PARAMS(args))
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:354:4: note: expanded from macro '_DEFINE_TRACEPOINT'
                        _TP_EXTRACT_STRING(_args),                              \
                        ^~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:340:33: note: expanded from macro '_TP_EXTRACT_STRING'
#define _TP_EXTRACT_STRING(...) #__VA_ARGS__
                                ^~~~~~~~~~~~
<scratch space>:71:26: note: expanded from here
"const unsigned long long\ Addressn const unsigned long long\ SizeR const unsigned inti Type"
                         ^~
In file included from /__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/eventprovdotnetruntime.cpp:24:
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:2821:1: error: unknown escape sequence '\ ' [-Werror,-Wunknown-escape-sequence]
GCCreateSegmentT_TRACEPOINT_INSTANCE(GCCreateSegment)
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:608:10: note: expanded from macro 'GCCreateSegmentT_TRACEPOINT_INSTANCE'
TRACEPOINT_EVENT_INSTANCE(\
         ^~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:513:2: note: expanded from macro 'T_EVENT_INSTANCE'
        _DEFINE_TRACEPOINT(provider, name, _TP_PARAMS(args))
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:354:4: note: expanded from macro '_DEFINE_TRACEPOINT'
                        _TP_EXTRACT_STRING(_args),                              \
                        ^~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:340:33: note: expanded from macro '_TP_EXTRACT_STRING'
#define _TP_EXTRACT_STRING(...) #__VA_ARGS__
                                ^~~~~~~~~~~~
<scratch space>:71:61: note: expanded from here
"const unsigned long long\ Addressn const unsigned long long\ SizeR const unsigned inti Type"
                                                            ^~
In file included from /__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/eventprovdotnetruntime.cpp:24:
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:2833:1: error: unknown escape sequence '\ ' [-Werror,-Wunknown-escape-sequence]
GCAllocationTick_V2T_TRACEPOINT_INSTANCE(GCAllocationTick_V2)
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:1740:32: note: expanded from macro 'GCAllocationTick_V2T_TRACEPOINT_INSTANCE'
#define AppDomainMemAllocated_TRACEPOINT_ARGS \
                               ^~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:513:2: note: expanded from macro 'RACEPOINT_ARGS'
        _DEFINE_TRACEPOINT(provider, name, _TP_PARAMS(args))
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:354:4: note: expanded from macro '_DEFINE_TRACEPOINT'
                        _TP_EXTRACT_STRING(_args),                              \
                        ^~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:340:33: note: expanded from macro '_TP_EXTRACT_STRING'
#define _TP_EXTRACT_STRING(...) #__VA_ARGS__
                                ^~~~~~~~~~~~
<scratch space>:13:171: note: expanded from here
"const unsigned int  AllocationAmounta const unsigned intu AllocationKinda const unsigned shorte ClrInstanceIDg const unsigned long longA AllocationAmount64N const size_t\ TypeIDd const char\
                                                                                                                                                                          ^~
In file included from /__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/eventprovdotnetruntime.cpp:24:
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:2833:1: error: unknown escape sequence '\xA' [-Werror,-Wunknown-escape-sequence]
GCAllocationTick_V2T_TRACEPOINT_INSTANCE(GCAllocationTick_V2)
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:1740:32: note: expanded from macro 'GCAllocationTick_V2T_TRACEPOINT_INSTANCE'
#define AppDomainMemAllocated_TRACEPOINT_ARGS \
                               ^~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:513:2: note: expanded from macro 'RACEPOINT_ARGS'
        _DEFINE_TRACEPOINT(provider, name, _TP_PARAMS(args))
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:354:4: note: expanded from macro '_DEFINE_TRACEPOINT'
                        _TP_EXTRACT_STRING(_args),                              \
                        ^~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:340:33: note: expanded from macro '_TP_EXTRACT_STRING'
#define _TP_EXTRACT_STRING(...) #__VA_ARGS__
                                ^~~~~~~~~~~~
<scratch space>:14:1: note: expanded from here
 TypeNamee const unsigned intf HeapIndex"
^~
In file included from /__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/eventprovdotnetruntime.cpp:24:
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:2851:1: error: unknown escape sequence '\ ' [-Werror,-Wunknown-escape-sequence]
GCGenerationRangeT_TRACEPOINT_INSTANCE(GCGenerationRange)
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:895:11: note: expanded from macro 'GCGenerationRangeT_TRACEPOINT_INSTANCE'
        const char*, EventName, \
          ^~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:513:2: note: expanded from macro 'nst'
        _DEFINE_TRACEPOINT(provider, name, _TP_PARAMS(args))
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:354:4: note: expanded from macro '_DEFINE_TRACEPOINT'
                        _TP_EXTRACT_STRING(_args),                              \
                        ^~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:340:33: note: expanded from macro '_TP_EXTRACT_STRING'
#define _TP_EXTRACT_STRING(...) #__VA_ARGS__
                                ^~~~~~~~~~~~
<scratch space>:69:21: note: expanded from here
"const unsigned char\ GenerationR const size_tn RangeStartd const unsigned long longn RangeUsedLength_ const unsigned long longk RangeReservedLength  const unsigned shortg ClrInstanceID"
                    ^~
[  1%] Building C object src/pal/src/libunwind/src/CMakeFiles/libunwind.dir/mi/dyn-register.c.o
Scanning dependencies of target coreclrtraceptprovider
[  1%] Building CXX object src/pal/src/eventprovider/lttngprovider/CMakeFiles/coreclrtraceptprovider.dir/lttng/traceptprovdotnetruntime.cpp.o
In file included from /__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/eventprovdotnetruntime.cpp:24:
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:2936:1: error: unknown escape sequence '\ ' [-Werror,-Wunknown-escape-sequence]
MethodJittingStartedT_TRACEPOINT_INSTANCE(MethodJittingStarted)
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:1635:1: note: expanded from macro 'MethodJittingStartedT_TRACEPOINT_INSTANCE'
        const unsigned int, MethodILSize, \
^       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:513:2: note: expanded from here
        _DEFINE_TRACEPOINT(provider, name, _TP_PARAMS(args))
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:354:4: note: expanded from macro '_DEFINE_TRACEPOINT'
                        _TP_EXTRACT_STRING(_args),                              \
                        ^~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:340:33: note: expanded from macro '_TP_EXTRACT_STRING'
#define _TP_EXTRACT_STRING(...) #__VA_ARGS__
                                ^~~~~~~~~~~~
<scratch space>:22:36: note: expanded from here
"const unsigned long long_ MethodID\ const unsigned long longn ModuleIDe const unsigned intM MethodTokenR const unsigned intt MethodILSizeR const charGS MethodNamespacet const char,  MethodNameo const char\
                                   ^~
In file included from /__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/eventprovdotnetruntime.cpp:24:
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:2936:1: error: unknown escape sequence '\xA' [-Werror,-Wunknown-escape-sequence]
MethodJittingStartedT_TRACEPOINT_INSTANCE(MethodJittingStarted)
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:1635:1: note: expanded from macro 'MethodJittingStartedT_TRACEPOINT_INSTANCE'
        const unsigned int, MethodILSize, \
^       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:513:2: note: expanded from here
        _DEFINE_TRACEPOINT(provider, name, _TP_PARAMS(args))
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:354:4: note: expanded from macro '_DEFINE_TRACEPOINT'
                        _TP_EXTRACT_STRING(_args),                              \
                        ^~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:340:33: note: expanded from macro '_TP_EXTRACT_STRING'
#define _TP_EXTRACT_STRING(...) #__VA_ARGS__
                                ^~~~~~~~~~~~
<scratch space>:23:1: note: expanded from here
 MethodSignature"
^~
[  1%] Building C object src/pal/src/libunwind/src/CMakeFiles/libunwind.dir/mi/Ldyn-extract.c.o
In file included from /__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/eventprovdotnetruntime.cpp:24:
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:2948:1: error: unknown escape sequence '\ ' [-Werror,-Wunknown-escape-sequence]
DomainModuleLoadUnload_V1T_TRACEPOINT_INSTANCE(DomainModuleLoad_V1)
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:2285:6: note: expanded from macro 'DomainModuleLoadUnload_V1T_TRACEPOINT_INSTANCE'
TRACEPOINT_EVENT_CLASS(
     ^~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:513:2: note: expanded from macro 'POINT_EVENT_CLASS'
        _DEFINE_TRACEPOINT(provider, name, _TP_PARAMS(args))
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:354:4: note: expanded from macro '_DEFINE_TRACEPOINT'
                        _TP_EXTRACT_STRING(_args),                              \
                        ^~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:340:33: note: expanded from macro '_TP_EXTRACT_STRING'
#define _TP_EXTRACT_STRING(...) #__VA_ARGS__
                                ^~~~~~~~~~~~
<scratch space>:8:217: note: expanded from here
"const unsigned long long_ ModuleIDn const unsigned long longn AssemblyID  const unsigned long long1 AppDomainID_ const unsigned intV ModuleFlags( const unsigned inth Reserved1a const char   ModuleILPatho const char \ ModuleNativePathL const unsigned shortA ClrInstanceID"
                                                                                                                                                                                                                        ^~
In file included from /__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/eventprovdotnetruntime.cpp:24:
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:2965:1: error: unknown escape sequence '\ ' [-Werror,-Wunknown-escape-sequence]
StrongNameVerification_V1T_TRACEPOINT_INSTANCE(StrongNameVerificationStart_V1)
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:2165:17: note: expanded from macro 'StrongNameVerification_V1T_TRACEPOINT_INSTANCE'
        ctf_string(MethodSignature, MethodSignature)
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:513:2: note: expanded from macro 'ng'
        _DEFINE_TRACEPOINT(provider, name, _TP_PARAMS(args))
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:354:4: note: expanded from macro '_DEFINE_TRACEPOINT'
                        _TP_EXTRACT_STRING(_args),                              \
                        ^~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:340:33: note: expanded from macro '_TP_EXTRACT_STRING'
#define _TP_EXTRACT_STRING(...) #__VA_ARGS__
                                ^~~~~~~~~~~~
<scratch space>:21:20: note: expanded from here
"const unsigned int\ VerificationFlagsi const unsigned int  ErrorCodec const char,  FullyQualifiedAssemblyNameo const unsigned shorte ClrInstanceID"
                   ^~
In file included from /__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/eventprovdotnetruntime.cpp:24:
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:2967:1: error: unknown escape sequence '\ ' [-Werror,-Wunknown-escape-sequence]
StrongNameVerification_V1T_TRACEPOINT_INSTANCE(StrongNameVerificationStop_V1)
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:2165:17: note: expanded from macro 'StrongNameVerification_V1T_TRACEPOINT_INSTANCE'
        ctf_string(MethodSignature, MethodSignature)
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:513:2: note: expanded from macro 'ng'
        _DEFINE_TRACEPOINT(provider, name, _TP_PARAMS(args))
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:354:4: note: expanded from macro '_DEFINE_TRACEPOINT'
                        _TP_EXTRACT_STRING(_args),                              \
                        ^~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:340:33: note: expanded from macro '_TP_EXTRACT_STRING'
#define _TP_EXTRACT_STRING(...) #__VA_ARGS__
                                ^~~~~~~~~~~~
<scratch space>:24:20: note: expanded from here
"const unsigned int\ VerificationFlagsi const unsigned int  ErrorCodec const char,  FullyQualifiedAssemblyNameo const unsigned shorte ClrInstanceID"
                   ^~
In file included from /__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/eventprovdotnetruntime.cpp:24:
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:2969:1: error: unknown escape sequence '\ ' [-Werror,-Wunknown-escape-sequence]
AuthenticodeVerification_V1T_TRACEPOINT_INSTANCE(AuthenticodeVerificationStart_V1)
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:2792:1: note: expanded from macro 'AuthenticodeVerification_V1T_TRACEPOINT_INSTANCE'
TRACEPOINT_EVENT_INSTANCE(\
^~~~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:513:2: note: expanded from macro 'TRACEPOINT_EVENT_INSTANCE'
        _DEFINE_TRACEPOINT(provider, name, _TP_PARAMS(args))
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:354:4: note: expanded from macro '_DEFINE_TRACEPOINT'
                        _TP_EXTRACT_STRING(_args),                              \
                        ^~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:340:33: note: expanded from macro '_TP_EXTRACT_STRING'
#define _TP_EXTRACT_STRING(...) #__VA_ARGS__
                                ^~~~~~~~~~~~
<scratch space>:28:78: note: expanded from here
 const unsigned intg ErrorCode4 const char   ModulePath  const unsigned short\ ClrInstanceID"
                                                                             ^~
In file included from /__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/eventprovdotnetruntime.cpp:24:
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:2971:1: error: unknown escape sequence '\ ' [-Werror,-Wunknown-escape-sequence]
AuthenticodeVerification_V1T_TRACEPOINT_INSTANCE(AuthenticodeVerificationStop_V1)
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/__w/2/s/bin/obj/Linux.arm.Release/src/pal/src/eventprovider/lttngprovider/lttng/tpdotnetruntime.h:2792:1: note: expanded from macro 'AuthenticodeVerification_V1T_TRACEPOINT_INSTANCE'
TRACEPOINT_EVENT_INSTANCE(\
^~~~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:513:2: note: expanded from macro 'TRACEPOINT_EVENT_INSTANCE'
        _DEFINE_TRACEPOINT(provider, name, _TP_PARAMS(args))
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:354:4: note: expanded from macro '_DEFINE_TRACEPOINT'
                        _TP_EXTRACT_STRING(_args),                              \
                        ^~~~~~~~~~~~~~~~~~~~~~~~~
/crossrootfs/arm/usr/include/lttng/tracepoint.h:340:33: note: expanded from macro '_TP_EXTRACT_STRING'
#define _TP_EXTRACT_STRING(...) #__VA_ARGS__
                                ^~~~~~~~~~~~
<scratch space>:33:78: note: expanded from here
 const unsigned intg ErrorCode4 const char   ModulePath  const unsigned short\ ClrInstanceID"
                                                                             ^~
[  1%] Building C object src/corefx/System.Globalization.Native/CMakeFiles/System.Globalization.Native.dir/__/__/__/version.c.o
[  1%] Linking C shared library System.Globalization.Native.so
[  1%] Building C object src/pal/src/libunwind/src/CMakeFiles/libunwind.dir/mi/Lfind_dynamic_proc_info.c.o
Verifying System.Globalization.Native dependencies
$<TARGET_FILE:System.Globalization.Native>.dbg
[  2%] Building C object src/pal/src/libunwind/src/CMakeFiles/libunwind.dir/mi/Lget_accessors.c.o
[  2%] Built target System.Globalization.Native
[  2%] Building CXX object src/pal/src/eventprovider/lttngprovider/CMakeFiles/coreclrtraceptprovider.dir/lttng/traceptprovdotnetruntimerundown.cpp.o
13 errors generated.
src/pal/src/eventprovider/lttngprovider/CMakeFiles/eventprovider.dir/build.make:100: recipe for target 'src/pal/src/eventprovider/lttngprovider/CMakeFiles/eventprovider.dir/lttng/eventprovdotnetruntime.cpp.o' failed
make[2]: *** [src/pal/src/eventprovider/lttngprovider/CMakeFiles/eventprovider.dir/lttng/eventprovdotnetruntime.cpp.o] Error 1
make[1]: *** [src/pal/src/eventprovider/lttngprovider/CMakeFiles/eventprovider.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

--- a/src/pal/src/eventprovider/lttngprovider/CMakeLists.txt
+++ b/src/pal/src/eventprovider/lttngprovider/CMakeLists.txt
@@ -40,16 +40,27 @@ add_custom_command(OUTPUT ${LTTNG_PROVIDER_SOURCES} ${TRACEPOINT_PROVIDER_SOURCE
   COMMAND ${GENERATE_COMMAND}
   DEPENDS ${EVENT_MANIFEST} ${GENERATE_SCRIPT})
 
+set_source_files_properties(
+  ${LTTNG_PROVIDER_SOURCES} ${TRACEPOINT_PROVIDER_SOURCES} ${LTTNG_HEADERS}
+  PROPERTIES GENERATED TRUE)
+
+add_custom_target(
+  generated_eventing_headers
+  DEPENDS ${LTTNG_HEADERS})
+
 add_library(eventprovider
-    STATIC
-    ${LTTNG_PROVIDER_SOURCES}
-    eventproviderhelpers.cpp
+  STATIC
+  ${LTTNG_PROVIDER_SOURCES}
+  eventproviderhelpers.cpp
 )
 
 add_library(coreclrtraceptprovider
-    SHARED
-    ${TRACEPOINT_PROVIDER_SOURCES}
+  SHARED
+  ${TRACEPOINT_PROVIDER_SOURCES}
 )
+
+add_dependencies(eventprovider generated_eventing_headers)
+add_dependencies(coreclrtraceptprovider generated_eventing_headers)
 
 find_library(LTTNG NAMES lttng-ust)
 

